src/components/organisms/BannerAdmin/BannerAdmin.tsx

```diff
@@ -1,89 +1,188 @@
+          />
+          {errors.content && errors.content.type === "required" && (
+            <span className="input-error">Display content is required</span>
+          )}
+        </div>
+
+        <Checkbox
+          containerClassName="BannerAdmin__checkbox BannerAdmin__checkbox--action"
+          labelClassName="BannerAdmin__checkbox__label"
+          name="isActionButton"
+          label="Call to Action Button"
+          toggler
+          forwardedRef={register}
+          defaultChecked={venue?.banner?.isActionButton}
+        />
+
+        <div className="BannerAdmin__action-container">
+          <InputField
+            ref={register}
+            name="buttonUrl"
+            placeholder="Button URL"
+            defaultValue={venue?.banner?.buttonUrl}
+            containerClassName="BannerAdmin__input-container"
+            inputClassName="BannerAdmin__input-text"
+            disabled={!isUrlButtonActive}
+            autoComplete="off"
+          />
+          <InputField
+            ref={register}
+            name="buttonDisplayText"
+            placeholder="Button display text"
+            defaultValue={venue?.banner?.buttonDisplayText}
+            containerClassName="BannerAdmin__input-container"
+            inputClassName="BannerAdmin__input-text"
+            disabled={!isUrlButtonActive}
+            autoComplete="off"
+          />
+        </div>
+        <Checkbox
+          forwardedRef={register}
+          containerClassName="BannerAdmin__checkbox"
+          labelClassName="BannerAdmin__checkbox__label"
+          name="isFullScreen"
+          label="Set fullscreen announcement"
+          toggler
+          defaultChecked={venue?.banner?.isFullScreen}
+        />
+
+        <Checkbox
+          forwardedRef={register}
+          containerClassName="BannerAdmin__checkbox"
+          labelClassName={forceFunnelLabelClasses}
+          name="isForceFunnel"
+          label="Force funnel (users will have to click your button)"
+          toggler
+          defaultChecked={venue?.banner?.isForceFunnel}
+          disabled={!isUrlButtonActive}
+        />
+
+        <div className="BannerAdmin__button-container">
+          <Button
+            customClass="BannerAdmin__button BannerAdmin__button--close"
+            onClick={clearBanner}
+          >
+            Clear
+          </Button>
+          <Button customClass="BannerAdmin__button" type="submit">
+            Save
+          </Button>
         </div>
-      </div>
+      </form>
+
+      <ConfirmationModal
+        show={isShowBannerChangeModal}
+        onConfirm={confirmChangeBannerData}
+        onCancel={hideBannerChangeModal}
+        header={"Erase your beautiful work?"}
+        message={"Are you sure?"}
+        saveBtnLabel="Clear & Save"
+        cancelBtnLabel="Cancel"
+        containerClassName={"ConfirmationBannerModal"}
+        headerClassName={"ConfirmationBannerModal__header"}
```

Comment:
These styles should be coming from `BannerAdmin.scss` and they should be named with the proper BEM syntax.

Reply:
isn't currently this component uses the same approach as other shared components? like passing class names into a component, so it defines styles on its own? otherwise we'll have to copy/paste this component's styles in other components should such an occasion happen
leaving it the way it is would make `ConfirmationModal` flexible for various styles

Reply:
fixed 947713e